tests/tasks/app/test_job_control.py

```diff
@@ -71,4 +69,107 @@ def test_asset_group_detection_jobs(
                 flask_app_client, staff_user, asset_group_uuid
             )
         tus_utils.cleanup_tus_dir(transaction_id)
-        assert orig_objs == test_utils.all_count(db)
+
+
+# Check that the task methods for the sighting job tasks print the correct output
+def test_sighting_identification_jobs(
+    flask_app,
+    flask_app_client,
+    researcher_1,
+    test_root,
+    db,
+):
+    # pylint: disable=invalid-name
+    from app.modules.sightings.models import Sighting, SightingStage
+
+    asset_group_uuids = []
+    sighting_uuids = []
+    transactions = []
+    try:
+        # Create two sightings so that there will be a valid annotation when doing ID for the second one.
+        # Otherwise the get_matching_set_data in sightings will return an empty list
```

Comment:
I forsee a testing util that ensures a matching set with a couple parameters in a future PR.